client/sections/services/sections.service.coffee

```diff
@@ -1,8 +1,11 @@
-Sections = new Mongo.Collection('sections')
-
 angular.module 'sections'
 .service 'sectionsServ', ($meteor) ->
     all: $meteor.collection(Sections).subscribe('sections')
 
     getTopics: (section_id) ->
```

Comment:
Zastanawiałem się też nad tym, ale to pobranie tematów w danym dziale, dlatego dałem tutaj. Na tej samej zasadzie jest getPosts w topicsServ